### ISSUE TITLE
The List-Unsubscribe header of the original email is lost

### ISSUE THREAD

**Issue by Fadenfire** [anonaddy/anonaddy#130]
Whenever an email is forwarded the List-Unsubscribe header is filled with an address that deactivates the alias. However if the the original email had a List-Unsubscribe header then the contents of the header are lost. I'm not sure what the best way to handle this is but here are two possibilities:
1. When the original message has a List-Unsubscribe header then copy it to the forwarded message. This would produce inconsistent behavior because unsubscribing from some messages would deactivate the alias and unsubscribing from others would unsubscribe from the original sender.
2. Put the unsubscribe address in the banner displayed at the top of the forwarded email. This probably the best option.

**Comment by mtalexan** [comment]
Much like the `Reply-To` versus `From` header conundrum, this seems like a candidate for a configurable setting.

While it's nice to provide an easy way to disable an alias, I suspect many people would prefer to preserve the `List-Unsubscribe` header. It seems like a special case behavior to disable an alias and not something you want accidentally triggered from intelligent email apps/providers when they run automatic unsubscribe actions on your behalf.

**Comment by willbrowningme** [comment]
Will add an option in settings to use the original list-unsubscribe header if one is set - https://github.com/anonaddy/anonaddy/projects/1#card-56423821

**Comment by ccorneli** [comment]
I just found this after replying to my original question in this thread #165 which is related. 

I have the banner turned off, so I don't have much use for option 2.

Option 1 is OK to preserve the original, and overwrite the alias unsubscribe,
but my main frustration is that ALL my email looks like bulk email, even when it is not. 
If the banner is disabled, I don't see a reason why the header should be included at all.
That way it's either omitted, or provided from the original sender.

**Comment by willbrowningme** [comment]
I've just pushed an update so that if there is a `List-Unsubscribe:` header present on the original email then it will be preserved when the email is forwarded. 

I've also removed the `List-Unsubscribe:` header that was added to every forwarded email that would allow you to deactivate that alias. The main reason being that a number of users were accidentally deactivating their aliases without realising, so I think it is best to remove it altogether for now.
